.github/workflows/verible-ci.yml

```diff
@@ -132,6 +132,7 @@ jobs:
       run: |
         set -x
         source ./.github/settings.sh
+        # dependencies provided by the container: gcc, m4, flex, bison
```

Comment:
Should we explicitly install flex and bison ? Even if they are by default on the CI system, this would document it.